packages/jaeger-ui/src/components/TraceDiff/TraceDiffHeader/TraceHeader.test.js

```diff
@@ -13,57 +13,77 @@
 // limitations under the License.
 
 import * as React from 'react';
-import { shallow } from 'enzyme';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
 
 import TraceHeader, { Attrs, EmptyAttrs } from './TraceHeader';
 import { fetchedState } from '../../../constants';
 
 describe('TraceHeader', () => {
-  const props = {
-    duration: 700,
-    error: { errorKey: 'errorValue' },
-    traceID: 'trace-id',
-    traceName: 'trace name',
-  };
-  let wrapper;
+  describe(`TraceHeader`, () => {
```

Comment:
what's the motivation for TraceHeader/TraceHeader nesting?

Reply:
There are three exported components in `TraceHeader`
- The main component `TraceHeader`
- Utility components `Attrs` and `EmptyAttrs`

So I thought it's better to nest all the `describe` block for each individual component under one main `describe`.

Reply:
The refactoring makes it much harder to evaluate the actual changes/fixes to the tests, can we split it into another PR?

Reply:
Sure. Will revert this nesting for now.